__tests__/unit/old-structure/tree-fetching/stratetegies/network-only.spec.ts

```diff
@@ -57,7 +57,7 @@ describe('Utils: tree fetching (load: network only)', () => {
     const params = { url, onChangeTree: jest.fn(), strategy, retry }
     await expect(viewClient.load(params)).rejects.toEqual([
       // @ts-ignore
-      new BeagleNetworkError(url),
+      new BeagleNetworkError(url,{} as Response,500 , 'GET'),
```

Comment:
```suggestion
      new BeagleNetworkError(url, {} as Response,500 , 'GET'),
```
It would be nice to add the space after the "," to keep a pattern.